src/crypto/cipher/index.js

```diff
@@ -1,80 +1,73 @@
-/**
- * @fileoverview Symmetric cryptography functions
- * @module crypto/cipher
- */
+import aes from './aes'; // can be imported dynamically once Web Crypto is used for AES-KW too
+import enums from '../../enums';
 
-import aes from './aes';
-import { DES, TripleDES } from './des';
-import CAST5 from './cast5';
-import TF from './twofish';
-import BF from './blowfish';
+export async function getCipher(algo) {
+  switch (algo) {
+    case enums.symmetric.aes128:
+    case enums.symmetric.aes192:
+    case enums.symmetric.aes256:
+      return aes(getCipherKeySize(algo));
```

Comment:
It seems a bit wasteful to create the AES constructor and methods each time AES is requested. So I would prefer to still make constants for `aes{128,192,256}` as before, and then return one of them as needed here.